Services/DeviceClient.cs

```diff
@@ -105,33 +108,10 @@ public class DeviceClient : IDeviceClient
             await this.SendRawMessageAsync(eventMessage);
         }
 
-        public Task UpdateTwinAsync(Device device)
+        public async Task UpdatePropertiesAsync(Device device)
```

Comment:
not sure if I'm missing something here. There is an `async` in the method signature, but the method body doesn't have an `await`, does it compile?